tests/api/printf.cpp

```diff
@@ -40,6 +40,15 @@ static void releaseStdout(int fd) {
     close(fd);
 }
 
+#define RETURN_ON_FAILURE(err, ret)                                            \
```

Comment:
This is similar to the main one. Was having some trouble using ASSERT_CL_SUCCESS in some cases so just copied and reused this.